templates/conf/kubernetes.conf.erb

```diff
@@ -17,11 +17,11 @@
   read_from_head true
 <% if is_v1 %>
   <parse>
-    @type json
+    @type "#{ENV['FLUENT_CONTAINER_TAIL_PARSER_TYPE'] || 'json'}"
     time_format %Y-%m-%dT%H:%M:%S.%NZ
   </parse>
 <% else %>
-  format json
+  format json_in_json
```

Comment:
Should be same as above?

Reply:
Oh, you are right, thank you! Just fixed it